doc/getting_started/index.ipynb

```diff
@@ -446,7 +446,35 @@
    "source": [
     "## Geographic plots\n",
     "\n",
-    "hvPlot can generate geographic plots and handle geographic data (e.g. GeoPandas DataFrame) thanks to [GeoViews](https://geoviews.org/) (optional dependency)."
+    "hvPlot can generate geographic plots and handle geospatial data (e.g., GeoPandas DataFrame) on its own. For more advanced mapping features, you can optionally install [GeoViews](https://geoviews.org/)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c226b5d7-8608-4998-b34a-064f7b21135c",
+   "metadata": {},
+   "source": [
+    "**Without GeoViews**"
```

Comment:
It'd be nice to quickly mention when geographic data can be displayed on tiles without having to install geoviews, which is when the data is in lat/long coordinates.